java/cdk8s-core/README.md

```diff
@@ -7,3 +7,6 @@ Example usage of core Kubernetes resources and how to define them within a cdk8s
 - `KubeService` (Core)
 - `KubeDeployment` (Core)
 
+### Setup
+1. [Install CDK8s CLI](https://cdk8s.io/docs/latest/getting-started/#install-the-cli)
+2. Execute ```mvn clean install``` to build the project.
```

Comment:
This will also run `cdk8s synth`? that seems off. I think we should either have:

```console
- mvn clean build
```

Or 

```console
- mvn clean install
- cdk8s synth
```

I think instructing the users to run `cdk8s synth` is more appropriate, even though a bit less java idiomatic i guess. But it exposes the cdk8s-cli, which we do want them to be aware of and know how to interact with.